packages/components/src/custom-select-control-v2/types.ts

```diff
@@ -92,7 +92,6 @@ type LegacyOnChangeObject = {
 };
 
 export type LegacyCustomSelectProps = {
-	children?: never;
```

Comment:
Removed this, since it was only necessary to make TypeScript pick the correct API version from the union type in the legacy adapter layer. (This was the [mutually exclusive key](https://github.com/maninak/ts-xor#description).)